### PR TITLE
Accessibility Recommendations

### PR DESCRIPTION
**Pull request by brynnelliott**
@kmomar @maneesha  

Here are my recommended changes. If you think they should be split up even more, let me know, I am happy to do that. 

##### colour 
Use adequate contrast to ensure the best readability. Colour should not be the only means of conveying information (indicating action, prompting a response, or distinguishing a visual element). Avoid using the colour combinations of red/green, green/brown, green/blue, blue/gray. Use shapes, patterns, textures, and labels to help distinguish differences. 

##### images
Include alt text on all images.

##### link text
Avoid "click here", "more", and "read more" when embedding a link in text. Use words that describe link, not the full URL. Link up to a full sentence but avoid using longer. Use non-colour indicators such as a persistent underline to highlight link. When linking an email, use Email first name last name, instead of just the person's name. 

##### plain language 
Where possible employ plain language and/or link to definitions of complex terms, like "global capacity" and "computational skills".

##### Videos 
All videos should be captioned. 

Added to inclusivity - in things to avoid
Sensory words (see hear, read, look, run, reach, listen)

Please delete this line and the text below before submitting your contribution.

---

### CODE REVIEW

**Comment by maneesha** [comment]
Thanks for putting this together @brynnelliott 
 I've made a few comments in the PR.

**Comment by maneesha** [comment]
Leaving note here that I'm setting time to review these changes with @brynnelliott together

File: topic_folders/communications/resources/style-guide.md [hunk 1]
```diff
@@ -82,6 +82,9 @@ But <br>
 
 The Carpentries, while sounding plural, is a singular entity. Therefore, when referring to The Carpentries, verbs should also be singular, _e.g.,_ ‘The Carpentries is …’. The full and correct title for the merged organisation is The Carpentries. However, terminology such as ‘A Carpentries workshop …’ is acceptable.
 
+##### colour 
+Use adequate contrast to ensure the best readability. Colour should not be the only means of conveying information (indicating action, prompting a response, or distinguishing a visual element). Avoid using the colour combinations of red/green, green/brown, green/blue, blue/gray. Use shapes, patterns, textures, and labels to help distinguish differences.
```
[the remark this answers is not in the document]
Comment (maneesha) [reply topic_folders/communications/resources/style-guide.md:86]: another comment (testing)

File: topic_folders/communications/resources/style-guide.md [hunk 2]
```diff
@@ -235,6 +248,9 @@ Please spell this out in text, _e.g.,_ not as %, and write as two words, not one
 ##### periods
 _See [full stops](#full-stops)._
 
+##### plain language 
+Where possible employ plain language and/or link to definitions of complex terms, like "global capacity" and "computational skills".
```
[the remark this answers is not in the document]
Comment (brynnelliott) [reply topic_folders/communications/resources/style-guide.md:252]: @maneesha I think plain language is important especially on our website. I know some of the lessons will be different and have specialized language. This was one of the recommendations from the Understood report. I think this needs to be a bigger discussion of how we want to handle it. I am happy to hold this one for a bit.

**Comment by maneesha** [comment]
Thank you @brynnelliott !

**Pull request merged**